src/User.php

```diff
@@ -1446,6 +1446,7 @@ public function syncLdapGroups()
                 if (count($authtype)) {
                     // Clean groups
                     $this->input["_groups"] = array_unique($this->input["_groups"]);
+                    $this->oldvalues["_ldap_groups"] = $this->input["_groups"];
```

Comment:
`oldvalues` is supposed to contains values that were present before the update. I do not think it is a good place to temporarly store input values.